app/assess/models/ui/applicants_response.py

```diff
@@ -40,7 +41,7 @@ def from_dict(cls, data: dict):
 
 
 @dataclass
-class QuestionAnswerPair(ApplicantResponseComponent):
+class QuestionAnswerPair(ApplicantResponseComponent, ABC):
```

Comment:
Whats the "ABC" here?

Reply:
This is an abstract class as it just contains shared state for the subsequent classes, there are no uses of the constructor here so thought I'd explicitly mark it as an abstract base class.  In retrospect I sort of wish I'd used just functions instead of inheritance, or maybe even protocols.

Reply:
https://docs.python.org/3/library/abc.html

Reply:
okay thanks, I like the consistent approach of using classes so no worries there 👍